runtime/stdlib/random.go

```diff
@@ -195,3 +205,204 @@ func NewRevertibleRandomFunction(generator RandomGenerator) StandardLibraryValue
 		},
 	)
 }
+
+// map for a quick type to byte-size lookup
+var typeToBytes = map[sema.Type]int{
```

Comment:
In the future it would be nice to replace this with a "bit-width" field on `sema.NumericType`

Reply:
https://github.com/onflow/cadence/pull/2986/commits/e18191c347180fce435f8bfa70604c4c429df477